synthesizer/src/store/helpers/memory/transition.rs

```diff
@@ -66,7 +67,6 @@ impl<N: Network> TransitionStorage<N> for TransitionMemory<N> {
             input_store: InputStore::open(dev)?,
             output_store: OutputStore::open(dev)?,
             finalize_map: MemoryMap::default(),
-            proof_map: MemoryMap::default(),
```

Comment:
should we mark the corresponding `DataID` as deprecated?

Reply:
Hmm good question, implemented it here: 50ab2d307
@raychu86 do we have a process in place for updating database objects? Should I document it in CONTRIBUTING?

Reply:
related: https://github.com/AleoHQ/snarkVM/pull/1560; also, only the `DataID` variant needs to remain (to not perturb the order of the others (unless we explicitly assign values to them)

Reply:
On advice from Howard, we're merging this PR into staging, allowing us to just remove the DataIDs